.github/workflows/update-package-lock.yaml

```diff
@@ -15,12 +15,14 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v3
+      with:
+        node-version: 14
 
     - name: Configure git and update package-lock.json
       run: |
         git config user.email "typescriptbot@microsoft.com"
         git config user.name "TypeScript Bot"
-        npm install --package-lock-only
+        npm install --package-lock-only --ignore-scripts

```

Comment:
Now that this is run with node 14, I think this can be dropped, though if it exists it probably won't be harmful. Shame about that npm bug.

Reply:
Note that this probably wouldn't work in npm7+ at all, because those versions don't just overwrite package-lock with new versions. When we switch to lockfile v2, I think we'll have to come up with a new updating system.